lua/lspkind/init.lua

```diff
@@ -1,185 +1,309 @@
-local lspkind = {}
+local M = {}
+local H = {}
 local fmt = string.format
 
-local kind_presets = {
-  default = {
--- if you change or add symbol here
--- replace corresponding line in readme
-    Text = "",
-    Method = "",
-    Function = "",
-    Constructor = "",
-    Field = "ﰠ",
-    Variable = "",
-    Class = "ﴯ",
-    Interface = "",
-    Module = "",
-    Property = "ﰠ",
-    Unit = "塞",
-    Value = "",
-    Enum = "",
-    Keyword = "",
-    Snippet = "",
-    Color = "",
-    File = "",
-    Reference = "",
-    Folder = "",
-    EnumMember = "",
-    Constant = "",
-    Struct = "פּ",
-    Event = "",
-    Operator = "",
-    TypeParameter = ""
+local kinds = {
+  -- Kind items order
+  order = {
+    cmp = {
+      Text = 1,
+      Method = 2,
+      Function = 3,
+      Constructor = 4,
+      Field = 5,
+      Variable = 6,
+      Class = 7,
+      Interface = 8,
+      Module = 9,
+      Property = 10,
+      Unit = 11,
+      Value = 12,
+      Enum = 13,
+      Keyword = 14,
+      Snippet = 15,
+      Color = 16,
+      File = 17,
+      Reference = 18,
+      Folder = 19,
+      EnumMember = 20,
+      Constant = 21,
+      Struct = 22,
+      Event = 23,
+      Operator = 24,
+      TypeParameter = 25,
+    },
+    sym = {
+      File = 1,
+      Module = 2,
+      Namespace = 3,
+      Package = 4,
+      Class = 5,
+      Method = 6,
+      Property = 7,
+      Field = 8,
+      Constructor = 9,
+      Enum = 10,
+      Interface = 11,
+      Function = 12,
+      Variable = 13,
+      Constant = 14,
+      String = 15,
+      Number = 16,
+      Boolean = 17,
+      Array = 18,
+      Object = 19,
+      Key = 20,
+      Null = 21,
+      EnumMember = 22,
+      Struct = 23,
+      Event = 24,
+      Operator = 25,
+      TypeParameter = 26,
+    },
   },
-  codicons = {
-    Text = "",
-    Method = "",
-    Function = "",
-    Constructor = "",
-    Field = "",
-    Variable = "",
-    Class = "",
-    Interface = "",
-    Module = "",
-    Property = "",
-    Unit = "",
-    Value = "",
-    Enum = "",
-    Keyword = "",
-    Snippet = "",
-    Color = "",
-    File = "",
-    Reference = "",
-    Folder = "",
-    EnumMember = "",
-    Constant = "",
-    Struct = "",
-    Event = "",
-    Operator = "",
-    TypeParameter = "",
+  -- Kind items list
+  list = {
+    cmp = require('vim.lsp.protocol').CompletionItemKind,
+    sym = require('vim.lsp.protocol').SymbolKind,
```

Comment:
Here should be _deepcopy_ because later `set_symbols` or some other function will try to modify this table.

Reply:
Done with https://github.com/onsails/lspkind.nvim/pull/47/commits/2583ba8de5a6d7ae3d784bcf8d4da39940be2cee